src/main/java/org/gridsuite/modification/server/dto/DeleteAttachingLineInfos.java

```diff
@@ -0,0 +1,76 @@
+/*
+  Copyright (c) 2022, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.dto;
+
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.ReporterModel;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.NetworkModificationException;
+import org.gridsuite.modification.server.entities.equipment.modification.DeleteAttachingLineEntity;
+import org.gridsuite.modification.server.modifications.AbstractModification;
+import org.gridsuite.modification.server.modifications.DeleteAttachingLine;
+
+/**
+ * @author bendaamerahm <ahmed.bendaamer at rte-france.com>
+ */
+@SuperBuilder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString(callSuper = true)
+@Schema(description = "Delete attaching line infos")
+public class DeleteAttachingLineInfos extends ModificationInfos {
+
+    @Schema(description = "line 1 id")
+    private String lineToAttachTo1Id;
+
+    @Schema(description = "line 2 id")
+    private String lineToAttachTo2Id;
+
+    @Schema(description = "attachment line id")
+    private String attachedLineId;
+
+    @Schema(description = "replacing line 1 ID")
+    private String replacingLine1Id;
+
+    @Schema(description = "replacing line 1 name")
+    private String replacingLine1Name;
+
+    @Override
+    public DeleteAttachingLineEntity toEntity() {
+        return DeleteAttachingLineEntity.toEntity(
+                getLineToAttachTo1Id(),
+                getLineToAttachTo2Id(),
+                getAttachedLineId(),
+                getReplacingLine1Id(),
+                getReplacingLine1Name()
+        );
```

Comment:
would be better to return new DeleteAttachingLineEntity(this);
This is what is done in other DTO (see LineAttachToVoltageLevel for instance)